HW-07/qw.py

```diff
@@ -0,0 +1,30 @@
+while True:
+    contacts = input('Enter contact: ')
+    con = contacts.split()
+
+    new_dic = {
+        'Frodo': '+12345678',
+        'Sam': '+87654321',
+        'Mari': '+18273645',
+    }
+
+    if contacts == 'list':
+        print(list(new_dic))
+
+    elif contacts == 'stats':
+        print('You have ' + str(len(new_dic)) + ' contacts.')
+    elif contacts == 'add':
+        new_contacts = input('Enter new contacts: ')
```

Comment:
It is not allowed to override an existing record by the task. Please, add an additional check here